### ISSUE TITLE
Integrate The Currency UI Into The Inventory UI

### ISSUE THREAD

**Issue by Brinklesby** [Drakensin/Dungeons-and-Delving#2]
The currency UI would be less clunky to access and look more attractive if it was integrated into the inventory UI

**Comment by ElPumpo** [comment]
Yes, we should ask the game devs for the real source code - so we can ditch these outdated decompilers
